Automattic/wp-calypso
# Tests: Update two broken VR test reference images

#### Changes proposed in this Pull Request

* Updates two screenshots for VR tests

#### Testing instructions

* Make sure tests pass in [TeamCity](https://teamcity.a8c.com/viewType.html?buildTypeId=calypso_VisualRegressionTests&branch_calypso_WPComTests=update%2Fvr-tests&tab=buildTypeStatusDiv)
